com.unity.cinemachine/Documentation~/CinemachineInputAxisController.md

```diff
@@ -28,3 +31,142 @@ If you like, you can also use CinemachineInputAxisController with your own scrip
+
+### Read from PlayerInput component
+
+To read values from a `PlayerInput` with a `behaviour` set to `InvokeCSharpEvents`, You need to create a custom `InputAxisController` that subscribes to `onActionTriggered`. The example bellow shows how to receive and wire those inputs accordingly. Add this script to your `CinemachineCamera` and assign the `PlayerInput` field.
+
+```cs
+using UnityEngine;
+using Unity.Cinemachine;
+using System;
+using UnityEngine.InputSystem;
+
+class PlayerInputReceiver : InputAxisControllerBase<SimpleReader>
+{
+    [SerializeField]
+    PlayerInput m_PlayerInput;
+
+    void Awake()
+    {
+        // Call back when the PlayerInput receives an Input.
+        m_PlayerInput.onActionTriggered += OnActionTriggered;
+    }
+
+    public void OnActionTriggered(InputAction.CallbackContext value)
+    {
+        // Sends the Input to all the controllers.
+        for (var i = 0; i < Controllers.Count; i ++)
+        {
+            Controllers[i].Input.SetValue(value.action);
+        }
+    }
+}
+
+[Serializable]
+class SimpleReader : IInputAxisReader
+{
+    // Assumes the action is a Vector2 for simplicity but can be changed for a float.
+    Vector2 m_Value;
+    [SerializeField]
+    InputActionReference m_Input;
+    [SerializeField]
+    float m_Gain;
+    
+    public void SetValue(InputAction action)
+    {
+        // Is the input referenced in the inspector matches the updated one update the value.
+        if (m_Input!= null && m_Input.action.id == action.id)
+            m_Value = action.ReadValue<Vector2>();
+    }
+
+    public float GetValue(UnityEngine.Object context, 
+        int playerIndex, 
+        bool autoEnableInput, 
+        IInputAxisOwner.AxisDescriptor.Hints hint)
+    {
+        if(hint == IInputAxisOwner.AxisDescriptor.Hints.X)
+            return m_Value.x * m_Gain;
+        
+        return m_Value.y * m_Gain;
+    }
+}
+
+#if UNITY_EDITOR
+[UnityEditor.CustomEditor(typeof(PlayerInputReceiver))]
+public class PlayerInputReceiverEditor : Unity.Cinemachine.Editor.InputAxisControllerEditor {}
+#endif
+```
+
+### Local split screen multiplayer
+
+In a split screen multiplayer environment using `PlayerInputManager` and `PlayerInput` components you will need one `CinemachineBrain` and one `CinemachineCamera` per player `Camera` and increase the channel mask of both the `CinemachineCamera` and the `CinemachineBrain`.
+
+```cs
+using Unity.Cinemachine;
+using UnityEngine;
+
+public class CinemachineCameraSetter : MonoBehaviour
+{
+    [SerializeField]
+    CinemachineBrain m_CinemachineBrain;
+    [SerializeField]
+    CinemachineCamera m_CinemachineCamera;
+
+    void Start()
+    {
+        // Places each new player in a line on the X axis.
+        transform.position = new Vector3(CinemachineCore.Instance.BrainCount, 2, 0);
```

Comment:
This line seems not relevant to what you're trying to show in this example.

Reply:
Fixed